port/unix/omrsysinfo.c

```diff
@@ -1718,7 +1727,12 @@ omrsysinfo_get_limit(struct OMRPortLibrary *portLibrary, uint32_t resourceID, ui
 	Trc_PRT_sysinfo_get_limit_Entered(resourceID);
 
 	if (OMRPORT_RESOURCE_ADDRESS_SPACE == resourceRequested) {
+#if !defined(OMRZTPF)
 		resource = RLIMIT_AS;
+#else /* !defined(OMRZTPF) */
+		resource = OMRPORT_LIMIT_UNKNOWN;
+		return resource;
```

Comment:
Updated set limit case to be consistent with get limit case.  Also added a comment saying z/TPF doesn't support virtual addressing.